### PR TITLE
Fixed indentations

### PR DESCRIPTION
**Pull request by nicalico**
Fixed whitespace/tabs indentations

### CODE REVIEW

File: api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddAttendeeRoleToPersonGenerator.java [hunk 1]
```diff
@@ -8,82 +8,84 @@
...
+        * Use the methods below to change the date/time precision in the
+        * custom form associated with this generator. When not used, the
+        * precision will be YEAR. The other precisons are MONTH, DAY, HOUR,
+        * MINUTE, TIME and NONE.
+        */
+    /*
         public String getStartDatePrecision() {
             String precision = VitroVocabulary.Precision.MONTH.uri();
-    	    return precision;
+            return precision;
```
Comment (awoods) [inline api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddAttendeeRoleToPersonGenerator.java]: It is only a minor difference, but I don't see this update in [Michel's original branch](https://github.com/vivo-project/VIVO/compare/sprint-i18n-whitespace?expand=1#diff-292a7b7c68e605ea463b9b9511cca2ceR81)

File: api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddAttendeeRoleToPersonGenerator.java [hunk 2]
```diff
@@ -8,82 +8,84 @@
...
         }
 
         public String getEndDatePrecision() {
             String precision = VitroVocabulary.Precision.DAY.uri();
-    	    return precision;
+            return precision;
```
Comment (awoods) [inline api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddAttendeeRoleToPersonGenerator.java]: It is only a minor difference, but I don't see this update in [Michel's original branch](https://github.com/vivo-project/VIVO/compare/sprint-i18n-whitespace?expand=1#diff-292a7b7c68e605ea463b9b9511cca2ceR86)

**Comment by nicalico** [comment]
Sorry. Fixed the line.

**Comment by nicalico** [comment]
Should be ok now. Didn't see it affected both lines. It seems likes my text editor was adding the whitespaces, and I had to vi the file to remove them.

**Pull request merged**